creator/files/migrations/0027_restore_old_file_types.py

```diff
@@ -0,0 +1,80 @@
+# Generated by Django 2.2.24 on 2021-09-10 21:07
+"""
+This migration does 2 things:
+    - Attempt to reverse the migration 0024_set_old_types_to_oth which changed
+    files with file type SHM and CLN to OTH
+    - Attempt to change the file type of all files marked as OTH to something
+    more specific
+"""
+
+from django.db import migrations, models
+
+
+def set_file_type(apps, schema_editor):
+    """
+    Try our best to set files with file_type OTH to a more specific file type
```

Comment:
Are there any risks to consider if a file gets misclassified?

Reply:
Yea I'm not sure. Probably a good question for @chris-s-friedman. When I ran the mapping function on prod data from Friday it seemed to work fairly well. I could have @chris-s-friedman take a look at the results of the data migration so we can be sure nothing gets misclassified?

Reply:
> I could have @chris-s-friedman take a look at the results of the data migration so we can be sure nothing gets misclassified? 

I can do that :)